src/GSF.CarbonAware/src/Handlers/CarbonAware/CarbonAwareParameters.cs

```diff
@@ -0,0 +1,378 @@
+using CarbonAware.Model;
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Json.Serialization;
+
+namespace GSF.CarbonAware.Handlers.CarbonAware;
+
+public class CarbonAwareParametersBaseDTO
```

Comment:
This is great! The current WebAPI PR uses the CarbonAware DTOs but depending on which PR gets merged first I'll update the WebAPI accordingly with this reference instead :)